build.gradle

```diff
@@ -76,6 +76,8 @@ project.ext.externalDependency = [
     // avro-serde includes dependencies for `kafka-avro-serializer` `kafka-schema-registry-client` and `avro`
     'kafkaAvroSerde': 'io.confluent:kafka-streams-avro-serde:5.5.1',
     'kafkaClients': 'org.apache.kafka:kafka-clients:2.3.0',
+    'kafkaAvroConnect': 'io.confluent:kafka-connect-avro-data:5.5.1',
```

Comment:
Are these needed?

Reply:
Yes, it is in these confluent modules that our serialization breaks when using confluent software (Kafka connect) to read our avro events.

I use them in the unit test, you can check there.